BookB-Pset_GeoScienceFeatureTypeFold

larswik opened the pull request:
# Pset: Pset_GeoScienceFeatureTypeFold

## Applicability: IfcGeoScienceFeature/FOLD

## Description: 

Properties for folds.

| Property | Datatype | Description |
|----------|----------|-------------|
|ProfileType|IfcIdentifier|Contains a term from a controlled vocabulary specifying the concave/convex geometry of fold relative to earth surface, and relationship to younging direction in folded strata if known. (e.g., antiform, synform, neutral, anticline, syncline, monocline, ptygmatic).|
|Strike|IfcPlaneAngleMeasure|The geographic direction of a line created by the intersection of a plane and the horizontal.|
|Quadrant|IfcPlaneAngleMeasure|The dip orientation. |
|DipDirection|IfcPlaneAngleMeasure|The geographic  direction of a line created by the intersection of a plane and the horizontal plane. If non specific convention is used, this angular value is in the range 0 to 18 degrees. In this case, there is an ambiguity on the dip orientation. For example, a plane with an orientation of 90 degrees from the nord could either have a dip direction to the North or to the South. Thus, to avoid this ambiguity, a strike value is generally completed with an indication of the dip orientation (Quadrant).|

larswik committed:
BookB-Pset_GeoScienceFeatureTypeFold

SergejMuhic commented on IFC4x3/Properties/q/Quadrant_2KShyZ_Xf6twsGaPpAX7HT/DocProperty.xml:
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Quadrant_2KShyZ_Xf6twsGaPpAX7HT" Name="Quadrant" UniqueId="9472bf23-fa1a-46df-ad90-919cca84745d" PrimaryDataType="IfcPlaneAngleMeasure" />
```
This property name and description do not seem to match.

larswik replied:
> This property name and description do not seem to match.

Agreed! @j-weil

larswik replied:
Removed. Didn't see the purpose with both DipDirection and StrikeDirection.

SergejMuhic commented on IFC4x3/Properties/d/DipDirection_17WEfvK3L9Ehv8Rdn5AK_c/DocProperty.xml:
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="DipDirection_17WEfvK3L9Ehv8Rdn5AK_c" Name="DipDirection" UniqueId="4780ea79-5035-493a-be48-6e7c45294fa6" PrimaryDataType="IfcPlaneAngleMeasure" />
```
Re-using, right?

SergejMuhic commented on IFC4x3/Properties/s/Strike_2QDp8uDwH5yOXoaLbKFWum/Documentation.md:
```diff
@@ -0,0 +1 @@
+The geographic direction of a line created by the intersection of a plane and the horizontal.
```
The angle is between true north and the created line? Measured CCW or CW?

pjanck replied:
Geographic directions are (usually?) measured CW from North.

SergejMuhic reviewed (commented):
With types, now it would probably make sense to use IfcTypeObject.

larswik committed:
Merge remote-tracking branch 'upstream/tunnel' into feature/BookB-Pset_GeoScienceFeatureTypeFold

larswik committed:
Merge remote-tracking branch 'upstream/tunnel' into feature/BookB-Pset_GeoScienceFeatureTypeFold

pjanck commented on IFC4x3/Properties/p/ProfileType_1AEKQiP7PFbuAGZ6obzmNO/Documentation.md:
```diff
@@ -0,0 +1 @@
+Contains a term from a controlled vocabulary specifying the concave/convex geometry of fold relative to earth surface, and relationship to younging direction in folded strata if known. (e.g., antiform, synform, neutral, anticline, syncline, monocline, ptygmatic).
```
```suggestion
Contains a term from a controlled vocabulary specifying the concave/convex geometry of fold relative to earth surface, and relationship to younging direction in folded strata if known (e.g., antiform, synform, neutral, anticline, syncline, monocline, ptygmatic).
```

larswik committed:
Merge remote-tracking branch 'upstream/tunnel' into feature/BookB-Pset_GeoScienceFeatureTypeFold

pjanck reviewed (approved):
Reviewed verbally with @larswik.

Note for the future: a unification of properties like Quadrant would be beneficial :)

pjanck committed:
Update IFC4x3/Properties/p/ProfileType_1AEKQiP7PFbuAGZ6obzmNO/Documentation.md

larswik committed:
Merge remote-tracking branch 'upstream/tunnel' into feature/BookB-Pset_GeoScienceFeatureTypeFold

larswik committed:
Strike=>StrikeDirection and removed Quadrant

We couldn't see the reason for Quadrant and furthermore it had a description which did not clarify the purpose. Will have to be re-added in some addendum if really needed.
